test/realtime/message.test.js

```diff
@@ -917,6 +917,75 @@ define(['ably', 'shared_helper', 'async', 'chai'], function (Ably, helper, async
       );
     });
 
+    it('subscribe_with_filter_object', function (done) {
```

Comment:
We should also have a test for unsubscribing with a filter object

Reply:
[Done](https://github.com/ably/ably-js/pull/1003/commits/3192a0bdede12e59e37edb4eda726dcf8e708f75)